Mathlib/GroupTheory/SpecificGroups/Alternating.lean

```diff
@@ -85,12 +99,16 @@ theorem finRotate_bit1_mem_alternatingGroup {n : ℕ} :
 
 end Equiv.Perm
 
+@[simp]
+theorem alternatingGroup_index [Nontrivial α] :
+    (alternatingGroup α).index = 2 := by
+  simp only [alternatingGroup, index, Nat.card_eq_fintype_card, ← Fintype.card_units_int]
+  exact Fintype.card_congr
+    (QuotientGroup.quotientKerEquivOfSurjective _ (sign_surjective α)).toEquiv
```

Comment:
Slightly shorter proof and the Subsingleton result [here](https://github.com/leanprover-community/mathlib4/commit/109d35c502ecb3c442470ddd7faf58ff89eaf110#diff-afd72e7846a1687d99079df9594f1ca23c815a0ff1fb47360980a668cde79410R358-R363).